test/over_react_redux/value_mutation_checker_test.dart

```diff
@@ -148,9 +147,6 @@ void sharedHashTests({bool useLengthHasher}) {
 
       map['newField'] = true;
       expect(hasher.hasHashChanged(map), isTrue);
-
-      map['test'] = false;
-      expect(hasher.hasHashChanged(map), isTrue);
```

Comment:
What was the reason for deleting this?

Reply:
Ah, I should have commented on this before, sorry!

This test suite was previously being run on the `CollectionShallowHasher` twice instead of once on it and once on `CollectionLengthHasher`, and this test was written in a way that it failed for `CollectionLengthHasher`. I assumed this was a mistake that just never got caught since tests weren't running properly, so I removed it.

And since what this expectation was exercising—that the hash changes when updating a value—is already being tested by the `hash` tests in the non-shared suite, I didn't think I needed to add any additional coverage.